### ISSUE TITLE
Mutation operation "REMOVE" for Campaign has incorrect typings

### ISSUE THREAD

**Issue by eliotik** [Opteo/google-ads-api#285]
In current version 5.2.0
Type for MutationOperation is same for all type of resources:
![image](https://user-images.githubusercontent.com/4066101/121747380-55b98b00-cad5-11eb-978b-032db618cd1b.png)
It's not compatible with https://developers.google.com/google-ads/api/reference/rpc/v7/CampaignOperation#remove

Instead of `delete` for operation it should be `remove`.
And since MutationOperation based on generic, and generic type used for `resource` property we specify operation as `MutateOperation<resources.ICampaign>`, this doesn't work for `remove` operation, because for removal `resource` should be of `string` type equal to `resource_name` path for campaign.

So operation object should be:
```
{
  // @ts-ignore
  operation: "remove",
  // @ts-ignore
  resource: 'customers/{customer_id}/campaigns/{campaign_id}',
  entity: "campaign"
} 
```

Otherwise we will get error:
```
{
    "error_code": {
        "request_error": "RESOURCE_NAME_MALFORMED"
    },
    "message": "Resource name '[object Object]' is malformed: expected 'customers/{customer_id}/campaigns/{campaign_id}'.",
    "location": {
        "field_path_elements": [
            {
                "field_name": "mutate_operations",
                "index": 4
            },
            {
                "field_name": "campaign_operation"
            },
            {
                "field_name": "remove"
            }
        ]
    }
},
```

or another one if we use `delete` instead of `remove`:
```
{
    "error_code": {
        "request_error": "OPERATION_REQUIRED"
    },
    "message": "Mutate operations must have 'create', 'update', or 'remove' specified.",
    "location": {
        "field_path_elements": [
            {
                "field_name": "mutate_operations",
                "index": 9
            },
            {
                "field_name": "campaign_operation"
            }
        ]
    }
},
```

**Comment by rosslavery** [comment]
I think this is a duplicate of https://github.com/Opteo/google-ads-api/issues/256

**Comment by WillCooter** [comment]
@eliotik @rosslavery  `delete` will be changed to `remove` in the next release of the API

**Comment by WillCooter** [comment]
@eliotik @rosslavery Thanks for your patience. This has been fixed in the latest version of the library (v8.0.0).
